cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -1323,4 +1324,46 @@ public static void create(StructuredTableAdmin tableAdmin) throws IOException {
       createIfNotExists(tableAdmin, IDENTITY_TABLE_SPEC);
     }
   }
+
+  /**
+   * Schemas for operation runs.
+   */
+  public static final class OperationRunsStore {
+    public static final StructuredTableId OPERATION_RUNS =
+        new StructuredTableId("operation_runs");
+
+    public static final String RUN_FIELD = "run";
+    public static final String NAMESPACE_FIELD = "namespace";
+    public static final String TYPE_FIELD = "type";
+    public static final String STATUS_FIELD = "status";
+    public static final String START_TIME_FIELD = "start_time";
+    public static final String UPDATED_AT_FIELD = "updated_at";
+    // contains serialized OperationRunDetail
+    public static final String RUN_DETAILS_FIELD = "run_details";
```

Comment:
`run_details` is redundant, can just be `details`

Reply:
Updated